### ISSUE TITLE
Nocode is working, but I don't want it to be working. Please fix.

### ISSUE THREAD

**Issue by bugsbycarlin** [kelseyhightower/nocode#1907]

**Comment by Yohann-Vila** [comment]
Please, why this issue isn't fixed already ?

**Comment by drawcard** [comment]
The solution I found on StackEchange is to hit the side of your monitor as hard as you can, repeatedly, while yelling at it until it stops working.

I have tested this solution and it checks out.

**Comment by bvanvliet** [comment]
Obscenities. You'll want to yell obscenities.

**Comment by bugsbycarlin** [comment]
That is a good idea. I combined both solutions and it worked. Using nocode's github API to close the issue.

**Comment by bvanvliet** [comment]
You're better off `NotUsing` Github's API. I took precisely _zero_ time yesterday to perform any benchmarking:

&nbsp; | Payload | Requests | Responses | Result
------------ | ------------ | ------------- | ------------- | -------------
**`Using Github API`** | 8mb | 15 | 15 | 27.0 `ms`
**`NotUsing API`** | &nbsp; | &nbsp; | &nbsp; | 0.00 `ms`

As you can see in the above table, comparing **NotUsing**  to using Github's API results in a statement which has _zero_ meaning, logical or otherwise. Another unsurprising result is the level of regret at having expended the amount energy, time, and brain that I have on this post; something that is _quite_ measurable.

So in conclusion. NotUsing NoCode has no linguistic meaning. This paragraph will not trail off int.....
